### ISSUE TITLE
Articles of Bylaws are typically designated by Roman numerals

### ISSUE THREAD

**Issue by goxberry** [jbardhan/csgf-draftdocs#6]
Articles of Bylaws are typically denoted by Roman numerals, rather than Arabic numerals (_Robert's Rules of Order Newly Revised (11th ed.)_, p. 570, ll. 23-25).

**Comment by jbardhan** [comment]
Noted.  They will be done. Couldn't figure out how to do that in this Wiki.  Thoughts?

**Comment by goxberry** [comment]
tl;dr: No idea. Looks like a minor issue, and I support the current organizational format.

---

I think the Roman numeral thing is just convention from _Robert's Rules_, which dates back to 1876. I can only speculate that the reasoning in having capital Roman numerals for articles, Arabic numerals for sections, and for subsequent levels of organization, maybe lowercase Roman numerals for subsections, and Latin letters for clauses is that it permits atomic reference of bylaws (III.4.i(a) for Article III, Section 4, Subsection i, clause (a)). The same thing can be achieved with the current conventional LaTeX decimal notation (for the previous example, 3.4.1(a), or 3.4.1a, since there isn't a need to disambiguate the 1st subsection from the 9th clause, both of which would be denoted by "i").
